src/components/select-field/BaseSelectField.js

```diff
@@ -352,7 +357,13 @@ class BaseSelectField extends React.Component<Props, State> {
                     <div className="select-option-check-icon">
                         {isSelected ? <IconCheck height={16} width={16} /> : null}
                     </div>
-                    {optionRenderer ? optionRenderer(item) : displayText}
+                    {optionRenderer ? (
+                        optionRenderer(item)
+                    ) : (
+                        <span className="select-option-text" title={displayText}>
```

Comment:
Can this be a the `defaultProp` value for `optionRenderer`?

Reply:
Makes sense!